src/lib/Scenes/Favorites/Components/Categories/index.tsx

```diff
@@ -15,43 +15,83 @@ export class Categories extends React.Component<RelayProps, null> {
       />
     )
 
+    const loadMore = () => {
+      if (!this.props.relay.hasMore() || this.props.relay.isLoading()) {
+        return
+      }
+
+      const updateState = (loading: boolean) => this.setState({ fetchingMoreData: loading })
```

Comment:
Is it really still necessary to set this state if apperantly the `relay` prop has an `isLoading` property?

Reply:
Great Q, I took this from the `Messages` component - I don't think it's being used anywhere actually TBH

Reply:
Yeah I think that can go then